substrate/frame/support/src/inherent.rs

```diff
@@ -81,6 +82,13 @@ pub trait ProvideInherent {
 		Ok(())
 	}
 
+	/// The order of the inherent relative to all other inherents.
+	///
+	/// A return value of `None` indicates that the pallet index should be used.
+	fn inherent_order() -> Option<InherentOrder> {
+		None
+	}
```

Comment:
Maybe 'priority' instead of 'order' is better description for the api.

So, inherents are 'ordered' by 'priority'. 

```suggestion
	fn inherent_priority() -> Option<InherentPriority> {
		None
	}
```